packages/cssnano-utils/src/__tests__/getMatch.test.js

```diff
@@ -1,17 +0,0 @@
-import { test } from 'uvu';
-import * as assert from 'uvu/assert';
-import getMatchFactory from '../getMatch';
-
-test('should get match', () => {
```

Comment:
You can see from this test that this code was meant to work like a custom map implementation.